docs/knowledgebase/advanced/ss58-address-format.md

```diff
@@ -10,6 +10,100 @@ The basic idea is a base-58 encoded value that can identify a specific account o
 chain. Different chains have different means of identifying accounts. SS58 is designed to be
 extensible for this reason.
 
-The living specification for the SS-58 address format can be found on the Substrate GitHub wiki:
+The living specification for the SS58 address format can be found on the Substrate GitHub wiki:
 
 https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)
+
+## Validating Addresses
+
+There are several ways to verify that a value is a valid SS58 address.
+
+### Subkey
+
+You can use the [Subkey](https://substrate.dev/docs/en/knowledgebase/integrate/subkey) `inspect`
+subcommand, which accepts the seed phrase, a hex-encoded address, or an SS58 address as the input
+URI. If the input is a valid address, it will return a list containing the corresponding public
+key (hex), account ID, and SS58 values.
+
+Subkey currently works for keys generated using: sr25519, ed25519, and secp256k1.
+
+> **NOTE:** If you input a valid SS58 value, Subkey will also return a network ID/version value
+> that indicates which network the address has been encoded for.
+
+```bash
+$ subkey inspect "12bzRJfh7arnnfPPUZHeJUaE62QLEwhK48QnH9LXeK2m1iZU"
+Public Key URI `12bzRJfh7arnnfPPUZHeJUaE62QLEwhK48QnH9LXeK2m1iZU` is account:
+  Network ID/version: polkadot
+  Public key (hex):   0x46ebddef8cd9bb167dc30878d7113b7e168e6f0646beffd77d69d39bad76b47a
+  Account ID:         0x46ebddef8cd9bb167dc30878d7113b7e168e6f0646beffd77d69d39bad76b47a
+  SS58 Address:       12bzRJfh7arnnfPPUZHeJUaE62QLEwhK48QnH9LXeK2m1iZU
+```
+
+### Polkadot.js
+
+For verifying an address in your JavaScript projects, you can utilize the functions built
+into the [Polkadot.js API](https://github.com/polkadot-js/api/), which supports ed25519 and
+sr25519 key types.
```

Comment:
1. I'm not sure they key types are relevant here.
2. Are you sure it does not support secp256k1?

Reply:
Yeah, I'm not the greatest on the cryptography algos, but yeah on second glance I believe it does. Removing the key types in any case.